src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(health) {
+    const FULL_HEALTH = 100;
+
+    this.health = health;
+
+    this.health = FULL_HEALTH;
```

Comment:
The logic related to health is not correct: first you set `this.health` to value from argument, and then rewrite it with default value. Instead of that, you should just define `FULL_HEALTH` as default value of function parameter.